mne/beamformer/tests/test_lcmv.py

```diff
@@ -746,56 +747,84 @@ def test_lcmv_reg_proj(proj, weight_norm):
         assert_allclose(stc_cov.data.std(), 0.12, rtol=0.1)
 
 
-@pytest.mark.parametrize('reg, weight_norm, use_cov, lower, upper', [
-    (0.05, 'unit-noise-gain', True, 96, 98),
+@pytest.mark.parametrize('reg, weight_norm, use_cov, depth, lower, upper', [
     # the 0 reg is not so stable, can produce a wide range of scores
-    (0.00, 'unit-noise-gain', True, 44, 90),
-    (0.05, 'nai', True, 96, 98),
-    (0.05, None, True, 96, 98),
-    (0.05, 'unit-noise-gain', False, 83, 86),
+    (0.00, 'unit-noise-gain', True, None, 44, 90),
+    (0.05, 'unit-noise-gain', True, None, 97, 98),
+    (0.05, 'nai', True, None, 96, 98),
+    (0.05, 'nai', True, 0.8, 96, 98),
+    (0.05, None, True, None, 74, 76),
+    (0.05, None, True, 0.8, 90, 93),  # depth improves weight_norm=None
+    (0.05, 'unit-noise-gain', False, None, 83, 86),
+    (0.05, 'unit-noise-gain', False, 0.8, 83, 86),  # depth same for wn != None
```

Comment:
(and this was an implicit/less strict test of equivalence that was there before)